connectors/src/connectors/snowflake/index.ts

```diff
@@ -73,6 +73,15 @@ export class SnowflakeConnectorManager extends BaseConnectorManager<null> {
     }
     const credentials = credentialsRes.value.credentials;
 
+    if (!isSnowflakeCredentials(credentials)) {
+      return new Err(
+        new ConnectorManagerError(
+          "INVALID_CONFIGURATION",
```

Comment:
nit: @fontanierh any reason why CONNECTORS ERRORS ARE IN CAPS ?

Reply:
not really

Reply:
IT LOOKS WEIRD INDEED

Reply:
I think enum values in all cap don't necessarily look weirder (I might even prefer it tbh), but it isn't consistent with the rest of the code

Reply:
It seems you resolved the comment and merged but didn't address the comment @albandum  ?

Reply:
Yep forgot to push the last commit 😓 
https://github.com/dust-tt/dust/pull/9441